lib/oauth2.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#Complete OAuth2.0
+#
+from httplib2 import Http
+from oauth2client import file, client, tools
+import yaml
+import json
+from imaplib import IMAP4_SSL
+def authentication():
```

Comment:
more whitespace in between things -- in general, you can use the yapf formatter on the files to get a good format